File: Sopra/finished_project/sopra-fs19-group03-client/src/components/shared/models/MessageContainer.js





class MessageContainer {
    constructor(type, message){
        this.type = type;
        this.message = message;
    }
    getType(){return this.type;}
    getMessage(){return this.message;}
    setType(type){this.type = type;}
    setMessage(message){this.message = message;}
}
export default MessageContainer;

File: Sopra/finished_project/sopra-fs19-group03-server/src/main/java/ch/uzh/ifi/seal/soprafs19/dto/incoming/MessageContainer.java
package ch.uzh.ifi.seal.soprafs19.dto.incoming;

import ch.uzh.ifi.seal.soprafs19.constant.MessageType;

public class MessageContainer{

    private MessageType type;

    private Message message;

    public MessageContainer(Message message, MessageType type) {
        this.message = message;
        this.type = type;
    }
    public MessageType getType() {
        return type;
    }

    public Message getMessage() {
        return message;
    }
}
